stapi-pydantic/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 ## [Unreleased]
 
+### Changed
+
+- Added some more models from **stapi-fastapi**, removed conformance urls ([#51](https://github.com/stapi-spec/pystapi/pull/51))
```

Comment:
`Added some more models from **stapi-fastapi**` is that part of this PR?

I can only find
- Move conformance uris back up into stapi-fastapi
- Change all stapi_pydantic imports to take directly from the top-level

Reply:
otherwise looks good to me